src/Zttp.php

```diff
@@ -54,6 +54,11 @@ function asFormParams()
         return $this->bodyFormat('form_params')->contentType('application/x-www-form-urlencoded');
     }
 
+    function asMultipart()
+    {
+        return $this->bodyFormat('multipart')->contentType("multipart/form-data; boundary=".md5(uniqid(null, true)));
```

Comment:
I tested it with multiple different HTTP Responses and sites like httpbin.org and the result is always a complain that the boundary is missing.